src/test/rustdoc/issue-75588.rs

```diff
@@ -0,0 +1,16 @@
+// aux-build:realcore.rs
+
+// Ensure unstably exported traits have their Implementors sections.
+
+#![crate_name = "foo"]
+#![feature(extremely_unstable_foo)]
+
+extern crate realcore;
+extern crate gimli;
+
+// issue #74672
+// @has foo/trait.Deref.html ''
```

Comment:
is this check correct for checking nothing in doc of Deref ?

Reply:
No, you should use `@!has` and then some condition. Not sure which would match here - I highly recommend writing this test against a commit _before_ your PR to make sure it fails before and succeeds after.

Reply:
I checked, the test failed without the fix.